zeppelin-server/src/main/java/org/apache/zeppelin/rest/message/NewNoteRequest.java

```diff
@@ -31,11 +31,16 @@
   //TODO(zjffdu) rename it to be notePath instead of name
   private String name;
   private String defaultInterpreterGroup;
+  private Boolean addingEmptyParagraph;
```

Comment:
Just use `boolean`, and initialize it with `false` by default.

Reply:
Sure, more elegant. Modified